### ISSUE TITLE
Acceptance Test for "New User Registration Page"

### ISSUE THREAD

**Issue by patrickjahns** [owncloud/core#32560]
With https://github.com/owncloud/core/pull/32466 ( and https://github.com/owncloud/user_management/pull/48 ) the feature was added, that users (created without password) will receive a notification mail to set their own password.

In the acceptance tests, we should test the following:
### First iteration
- [x] webUI  #32638 https://github.com/owncloud/user_management/pull/65

### Second Iteration ( once the first iteration is done and the behavior is established )
- [ ] provisioning api
- [ ] CLI

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks the contributor most likely able to help you is @phil-davis.

Possibly related issues are https://github.com/owncloud/core/issues/22485 (test), https://github.com/owncloud/core/issues/22492 (test), https://github.com/owncloud/core/issues/31579 (test), https://github.com/owncloud/core/issues/31116 (Acceptance tests for lost password email), and https://github.com/owncloud/core/pull/32312 (check if user/group were created in acceptance tests).

**Comment by individual-it** [comment]
This functionality is not implemented for CLI and provisioning api, in both cases a valid password must be provided.
@sharidas can you confirm that this new functionality is only to be used in the webUI ?

**Comment by sharidas** [comment]
It is yet to be extended for the commands like `user:add`. As of now I have made it for the UI.

**Comment by phil-davis** [comment]
I think an issue was raised for adding this to the CLI (or am I imagining?). A link to it from here would be useful.

**Comment by individual-it** [comment]
closing this issue as there is no test to write anymore. we should open a new issue when the functionality is extended
